lib/whitehall/exporters/documents_info_exporter.rb

```diff
@@ -7,10 +7,12 @@ def initialize(document_ids)
 
   def call
     document_ids.map do |doc_id|
+      document = Document.find(doc_id)
```

Comment:
Instead of looking up each Document from its ID here, would it be nicer to update `paginated_document_ids` in `app/controllers/admin/export/document_controller.rb` to return the documents (instead of the IDs)?

E.g. replace `pluck(:document_id)` with `.map(&:document)` at https://github.com/alphagov/whitehall/blob/d86695f2a60b6f204ab92bb98ec9968599d5f66c/app/controllers/admin/export/document_controller.rb#L63

This will reduce the number of database queries we need to do.  Although you would need to rename a few methods.

This method would then become:
```
  def call
    documents.map do |document|
      {
        document_id: document.id,
        document_information: {
          locales: locales_hash[doc_id],
          content_id: document.content_id,
        },
      }
    end
  end
```